setup.py

```diff
@@ -215,7 +215,7 @@ def get_extension_modules():
       author_email = 'michael.koenig@blue-yonder.com',
       packages = ['turbodbc'],
       extras_require={
-          'arrow': ['pyarrow>=0.15,<2.0'],
```

Comment:
Please put an upper limit here. Arrow breaks regularly and this will just confuse users when it doesn't work just because of an incompatible Arrow version. I would feel more comfortable doing regular releases than having to deal with all the user reports.

Reply:
Done, is the `<3.1.0` fine for you?

Reply:
Yes!